sdk/search/Azure.Search/src/Utilities/SearchExtensions.cs

```diff
@@ -47,5 +48,25 @@ public static string GetSearchServiceName(this Uri endpoint)
                 host.Substring(0, separator) :
                 null;
         }
+
+        /// <summary>
+        /// Join a collection of strings into a single comma separated string.
+        /// If the collection is null or empty, a null string will be returned.
+        /// </summary>
+        /// <param name="items">The items to join.</param>
+        /// <returns>The items joined together by commas.</returns>
+        public static string CommaSeparate(this ICollection<string> items) =>
```

Comment:
CommaJoin?

Reply:
I think `Separate` is pretty typical for this one - It's the `S` in `CSV`.  It's the `CommaSplit` name I didn't like as much.  😄

Reply:
I actually like Pavel's suggestion here. 'Join' is consistent with `string` and LINQ, and is the antonym of `CommaSplit` below so it's more discoverable - though, the audience is limited.

Reply:
Yeah, that's fair.  I've made the change.